OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -39,6 +39,18 @@ public interface IPlaceBuildingDecorationInfo : ITraitInfo
 	}
 
 	[RequireExplicitImplementation]
+	public interface IBlocksProjectiles
+	{
+		WDist BlockingHeight { get; }
+	}
+
+	[RequireExplicitImplementation]
+	public interface IBlocksProjectilesInfo : ITraitInfo
+	{
+		WDist MaximumBlockingHeight { get; }
```

Comment:
Ok, so it turns out that we aren't using this at all.  We should remove `MaximumBlockingHeight` until we have a use for it, keeping the interface as just a tag for your queries.